public/profile.html

```
@@ -1,45 +1,324 @@
+          <p class="text-xs text-gray-600 dark:text-gray-300 italic mt-2">Public profiles share your bio, expertise, and username. Real name and email remain private.</p>
+          <div id="public-link-row" class="hidden mt-3 items-center gap-2 p-2.5 bg-teal-50 dark:bg-teal-900/30 rounded-lg border border-teal-200 dark:border-teal-800">
+            <i class="fas fa-external-link-alt text-teal-500 flex-shrink-0 text-xs"></i>
+            <a id="public-link" href="#" target="_blank" class="text-xs text-teal-700 dark:text-teal-300 hover:underline truncate"></a>
+          </div>
+        </div>
+
+        <div class="grid grid-cols-1 sm:grid-cols-2 gap-4">
+          <div><label class="block text-sm font-medium mb-1" for="name">Display Name</label><input id="name" name="name" maxlength="80" class="w-full rounded-lg border border-gray-300 dark:border-gray-700 bg-white dark:bg-gray-900 px-3 py-2"></div>
+          <div>
+            <label class="block text-sm font-medium mb-1">Username</label>
+            <div class="relative"><span class="absolute left-3 top-1/2 -translate-y-1/2 text-gray-400 text-sm">@</span><input id="username-display" type="text" disabled class="w-full rounded-lg border border-gray-300 dark:border-gray-700 bg-gray-100 dark:bg-gray-800 text-gray-500 pl-7 pr-3 py-2 cursor-not-allowed"></div>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)